index.js

```diff
@@ -131,12 +136,7 @@ module.exports = {
       },
 
       _hasPackage(pkgName) {
-        try {
-          this.project.require.resolve(pkgName);
```

Comment:
Ember CLI seems to just offer `require`, without any of its members that you would normally expect. So `require.resolve()` was always throwing, leading to the function returning false.